security-ldap/src/main/java/io/micronaut/configuration/security/ldap/LdapAuthenticationProviderFactory.java

```diff
@@ -38,16 +42,18 @@
      * @param contextBuilder              The context builder
      * @param contextAuthenticationMapper The authentication mapper
      * @param ldapGroupProcessor          The group processor
+     * @param executorService             Executor Service
      * @return an {@link LdapAuthenticationProvider} if the corresponding {@link LdapConfiguration} is enabled
      */
     @EachBean(LdapConfiguration.class)
     public LdapAuthenticationProvider ldapAuthenticationProvider(@Parameter LdapConfiguration configuration,
                                                                            LdapSearchService ldapSearchService,
                                                                            ContextBuilder contextBuilder,
                                                                            ContextAuthenticationMapper contextAuthenticationMapper,
-                                                                           LdapGroupProcessor ldapGroupProcessor) {
+                                                                           LdapGroupProcessor ldapGroupProcessor,
+                                                                 @Named(TaskExecutors.IO) ExecutorService executorService) {
```

Comment:
Spacing on this is off

Reply:
I've fixed the spacing. The spacing was broken before this PR.